src/DotNetTwitterBot/Function.cs

```diff
@@ -43,6 +46,13 @@ public async Task Retweet(ILambdaContext context)
                     if (!tweet.Text.Contains(term, StringComparison.OrdinalIgnoreCase))
                         continue;
 
+                    // Exclude tweets that contain excluded words.
+                    if (filterTerms.Any(d => tweet.Text.Contains(d)))
+                    {
+                        //This sends the questionable tweet to the DM's of the me.User
+                        Message.PublishMessage($"Questionable Tweet : {tweet.Url}", User.GetUserFromScreenName($"{me.ScreenName}").Id);
```

Comment:
Sorry, missed this before. Is there an async version of `PublishMessage`? I would prefer to use that:
```csharp
await Message.PublishMessageAsync(...);
```

Reply:
no problem

Reply:
done